.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,49 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Building the Docker image
+      run: docker build . --file Dockerfile --tag my-image-name:$(date +%s)
```

Comment:
This step is not necessary, the image is being build on the `build-push-action@v2`

Reply:
That's correct, I have removed that step. Thanks!